src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne
+    @MapsId
+    private User user;
```

Comment:
I'm not sure, but it seems to me that one user can have many orders. At least that's how I have done it. We can wait for the mentor's review